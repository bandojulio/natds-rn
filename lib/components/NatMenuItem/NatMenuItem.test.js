var _jsxFileName='lib/components/NatMenuItem/NatMenuItem.test.js';var _react=require('react');var _react2=_interopRequireDefault(_react);
var _reactTestRenderer=require('react-test-renderer');var _reactTestRenderer2=_interopRequireDefault(_reactTestRenderer);
require('jest-styled-components/native');

var _NatMenuItem=require('./NatMenuItem');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}

describe('NatMenuItem component default',function(){
it('Should render NatMenuItem NatMenuItem ',function(){
var button=_reactTestRenderer2.default.create(
_react2.default.createElement(_NatMenuItem.NatMenuItem,{title:'Teste',__source:{fileName:_jsxFileName,lineNumber:10}})).
toJSON();

expect(button).toMatchSnapshot('NatMenuItem snapshot');
});
});
//# sourceMappingURL=NatMenuItem.test.js.map