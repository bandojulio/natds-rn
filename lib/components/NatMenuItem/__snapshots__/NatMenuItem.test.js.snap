// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NatMenuItem component default Should render NatLogoVertical NatMenuItem  1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 48,
      "justifyContent": "space-between",
      "marginLeft": 16,
      "opacity": 1,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../lib/assets/icons/nat-ic-arrow-left.png",
      }
    }
    style={
      Array [
        Object {
          "height": 24,
          "width": 24,
        },
      ]
    }
  />
  <Text
    style={
      Array [
        Object {
          "alignSelf": "center",
          "color": "#333333",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "marginLeft": 32,
        },
      ]
    }
    type="default"
  >
    Teste
  </Text>
</View>
`;
